src/t2iapi/biceps/metadata.proto

```diff
@@ -0,0 +1,43 @@
+/*
+This Source Code Form is subject to the terms of the MIT License.
+Copyright (c) 2024 Draegerwerk AG & Co. KGaA.
+
+SPDX-License-Identifier: MIT
+*/
+
+syntax = "proto3";
+
+package t2iapi.biceps;
+
+option java_package = "com.draeger.medical.t2iapi.biceps";
+option java_outer_classname = "MetadataProto";
+
+import "t2iapi/biceps/localizedtext.proto";
+import "t2iapi/biceps/instanceidentifier.proto";
+import "google/protobuf/wrappers.proto";
+
+
+
+
```

Comment:
```suggestion

```

Reply:
Weird diff, I just wanted to remove the blank lines

Reply:
done